### PR TITLE
tags: Added 'powercolor' tag.

### PR DESCRIPTION
**Pull request by ls-**
Addresses request https://github.com/oUF-wow/oUF/issues/247.

### CODE REVIEW

File: elements/tags.lua [hunk 1]
```diff
@@ -311,6 +311,25 @@ local tagStrings = {
...
 	end]],
+
+	['powercolor'] = [[function(u)
+		local pType, pToken, altR, altG, altB = UnitPowerType(u)
+		local t = _COLORS.power[pToken]
+
+		if(not t) then
+			if(altR) then
+				if(altR > 1 or altG > 1 or altB > 1) then
+					t = {altR / 255, altG / 255, altB / 255}
```
Comment (p3lim) [inline elements/tags.lua]: Return the value directly here, no need to create additional tables.
```Lua
return Hex(altR / 255, altG / 255, altB / 255)
```

File: elements/tags.lua [hunk 2]
```diff
@@ -311,6 +311,25 @@ local tagStrings = {
...
+				else
+					t = {altR, altG, altB}
```
Comment (p3lim) [inline elements/tags.lua]: Same as the comment above.

**Comment by ls-** [comment]
Logic failed me, yet again... 😕

**Comment by Rainrider** [comment]
You should also take power.GetAlternativeColor into account, so that the user could match the bar colours if they use it.

**Comment by ls-** [comment]
@Rainrider, `GetAlternativeColor` is power bar's method defined in layout. You can't access such things from inside tag function O_o

**Pull request merged**